tools/mozilla/buildxpi.py

```diff
@@ -50,7 +50,6 @@
 from subprocess import PIPE, CalledProcessError, Popen
 from tempfile import mkdtemp
 
-
```

Comment:
Please keep 2 blank lines after imports.

Reply:
There are no configurations in black.

Reply:
...but this was rather by non inteded changes in isort configuration. Fixed.